ext/standard/tests/dir/chdir_variation2-win32-mb.phpt

```diff
@@ -72,10 +73,11 @@ rmdir("$file_path/私はガラスを食べられますlevel_one");
 ===DONE===
```

Comment:
The two `rmdir("$file_path/…")` above should be `rmdir("$base_dir_path/…")`

Reply:
Should this code be there at all? Isn't this handled by the CLEAN section?

Reply:
Indeed, the code is superfluous!